components/butterfly/src/server/mod.rs

```diff
@@ -870,8 +878,10 @@ impl Server {
         e.term = term;
         let ek = RumorKey::from(&e);
         if !self.check_quorum(e.key()) {
+            warn!("start_election check_quorum failed: {:?}", e);
             e.no_quorum();
         }
+        debug!("start_election: {:?}", e);
```

Comment:
Can we add some information to the logging to indicate which service group we're doing an election for, or is that already taken into account?

Reply:
That's part of what gets dumped by the `Debug` trait on `Election`. For example:
```
Election { from_id: "d19c9c9a31f84c68a2e081dbb58d2770", member_id: "d19c9c9a31f84c68a2e081dbb58d2770", service_group: ServiceGroup("postgresql.staging"), term: 1, suitability: 83887240, status: Running, votes: ["d19c9c9a31f84c68a2e081dbb58d2770", "a0a6869826124dc18a6d538f015160a9", "7b2f61063fbc4775bb0727540b86ad6b"] } Election { from_id: "d19c9c9a31f84c68a2e081dbb58d2770", member_id: "d19c9c9a31f84c68a2e081dbb58d2770", service_group: ServiceGroup("postgresql.staging"), term: 1, suitability: 83887240, status: Running, votes: ["d19c9c9a31f84c68a2e081dbb58d2770", "a0a6869826124dc18a6d538f015160a9", "7b2f61063fbc4775bb0727540b86ad6b"]
```